web-client/src/pages/Home/usePreviewState.js

```diff
@@ -1,94 +1,33 @@
-import { computed, ref, reactive } from 'vue'
-import { TIERS } from '../../constants'
+import { computed, ref, watchEffect } from 'vue'
+import useTiers from './useTiers'
 
 export default function usePreviewState() {
-    // const variants = ref([
-    //     { name: 'Bronze', price: 9 },
-    //     { name: 'Silver', price: 29 },
-    //     { name: 'Gold', price: 79 },
-    //     { name: 'Diamond', price: 199 },
-    //     { name: 'Platinum', price: 499 },
-    // ])
-
-    const selected = ref(TIERS[1])
+    const tiers = useTiers()
+    const selectedName = ref('bronze')
+    const selected = computed(() => {
+        return selectedName.value
+            ? tiers.value.find((tier) => tier.name === selectedName.value)
+            : null
```

Comment:
Should `null` be a default of `steel` instead?

Maybe also `steel` a couple of lines up instead of `bronze`? (albeit changing to `steel` doesn't matter much since gets assigned here by `selected`.

Reply:
Yeah actually this one shouldn't matter as selectedName should never be NULL in the first place. 

That said you're right. Could be simplified and still produce the same outcome by: 

```
return tiers.value.find((tier) => tier.name === selectedName.value)
```

> Maybe also steel a couple of lines up instead of bronze? 
So my thought was to preselect a "most popular option" which most people can afford but isn't the cheapest. Also on desktop this would be the "middle tier" of the top row.  

> (albeit changing to steel doesn't matter much since gets assigned here by selected.
So actually this one here derives from `selectedName`. So it is in fact the default option that is set a couple lines up. For that reason too we don't need to set any default option here, if that makes sense

Reply:
Ah, good point about pre-selecting a "most popular option", nice!